frontend/src/diagram/reducer.ts

```diff
@@ -207,8 +207,8 @@ const handleDataAction = (prevState, action) => {
   const { message } = action;
 
   let state = prevState;
-  if (message.payload && message.payload.data && message.payload.data.diagramEvent) {
-    const { diagramEvent } = message.payload.data;
+  if (message.data && message.data.diagramEvent) {
+    const { diagramEvent } = message.data;
```

Comment:
No unit tests to update?

Reply:
Fixed by https://github.com/eclipse-sirius/sirius-components/pull/128/commits/0f6add0edf420bcbba4c3a325b7927a3c182746f